test/test_profile_mp_mnist.py

```diff
@@ -64,7 +64,11 @@ def _train_update(device, x, loss, tracker, writer):
       summary_writer=writer)
 
 
-def train_mnist(flags, **kwargs):
+def train_mnist(flags,
```

Comment:
I like the `**kwargs` + `.pop()` style better for a list of kwargs that can potentially grow too large, but this is fine.